ricoconverter/ricoconverter-convert/src/main/resources/xslt_eac/eac2rico.xslt

```diff
@@ -767,11 +767,23 @@
         </xsl:element>
 
 		<!-- shortcut property -->
-		<xsl:element name="{$HIERARCHICAL_RELATION_CONFIG/*[local-name() = normalize-space($type)]/shortcutIfSubjectIsSourceOfRelation}">
-        	<xsl:call-template name="rdf-resource">
-        		<xsl:with-param name="uri" select="eac2rico:URI-AgentExternal(@xlink:href, document(concat($INPUT_FOLDER, '/', @xlink:href, '.xml')))"/>	
-        	</xsl:call-template>
-        </xsl:element>
+		<xsl:variable name="shortcutProperty">
+			<xsl:choose>
+				<xsl:when test="not(eac:dateRange/eac:toDate)">
+					<xsl:value-of select="$HIERARCHICAL_RELATION_CONFIG/*[local-name() = normalize-space($type)]/shortcutIfSubjectIsSourceOfRelation_current" />
+				</xsl:when>
+				<xsl:otherwise>
+					<xsl:value-of select="$HIERARCHICAL_RELATION_CONFIG/*[local-name() = normalize-space($type)]/shortcutIfSubjectIsSourceOfRelation" />
```

Comment:
or `<xsl:value-of select="$HIERARCHICAL_RELATION_CONFIG/*[local-name() = normalize-space($type)]/shortcutIfSubjectIsSourceOfRelation_past" />`? See my comment on the eac2rico-relations.xslt